### PR TITLE
allow students to submit assessment without questions

### PR DESCRIPTION
**Pull request by huangjosh**

### CODE REVIEW

**Comment by allenwq** [comment]
@huangjosh Should we keep the grading panel same as before so that lecturers can assign exp ?

File: client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEmptyForm.jsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    this.state = {
+      unsubmitConfirmation: false,
+    };
+  }
+
+  submitAndRedirect = () => {
+    const { handleSubmit, courseId } = this.props;
+    handleSubmit().then(() => {
+      return history.push(
+        `/courses/${courseId}/assessments`
```
Comment (allenwq) [inline client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEmptyForm.jsx]: better redirect to the same tab and category.

**Comment by huangjosh** [comment]
screen shot 2017-09-11 at 6 17 59 pm

@allenwq The image above is the screenshot. 
Your comments are fixed.

**Comment by fonglh** [comment]
Fix the linting errors too
https://travis-ci.org/Coursemology/coursemology2/jobs/274134045#L1467

**Comment by huangjosh** [comment]
@fonglh Fixed, and updated the PR.

**Comment by huangjosh** [comment]
@fonglh Fixed and updated the PR.

**Pull request merged**